lib/index.js

```diff
@@ -7,6 +6,8 @@ var apiResponse = null;
 var weatherForecastObj = null;
 var userApiKey = null;
 var userFavoritesObj = null;
+var favDropDownLink = document.getElementById("dropDownFavs");
```

Comment:
I'd be careful with setting this variable set with `var` here. It is currently scoped globally and that can be an issue with the asynchronous nature of JS. Try to use `let` more as that scopes it locally. Look at [this Stack Overflow answer](https://stackoverflow.com/a/11444416) for more information on that